x/README.md

```diff
@@ -41,7 +41,7 @@ capabilities of your blockchain or further specialize it.
 ## Deprecated Modules
 
 The following modules are deprecated.  They will no longer be maintained and eventually will be removed
-in an upcoming release of the Cosmos SDK per our [release process](../RELEASE_PROCESS.md).
+in an upcoming release of the Cosmos SDK per our [release process](https://github.com/cosmos/cosmos-sdk/blob/main/RELEASE_PROCESS.md).
```

Comment:
changing this to a link because if i do a doc link, it puts `RELEASE_PROCESS` in the `modules` section of the doc, which feels messy, and consequently importing `release_process.md` also requires us to put `contributing.md` in the doc... 

i could just remove this, but i think linking to the main version of release process is the best way to get rid of this bug.